gatsby-config.js

```diff
@@ -539,5 +411,136 @@ module.exports = {
+            'erlang',
+            'gettext',
+            'pascal',
+            'parser',
+            'nginx',
+            'n1ql',
+            'monkey',
+            'mongodb',
+            'liquid',
+            'json5',
+            'jsdoc',
+            'mustache',
+            'powershell',
+            'promql',
+            'protobuf',
+            'puppet',
+            'jsx',
+            'regex',
+            'ruby',
+            'scala',
+            'shell',
+            'swift',
+            'systemd',
+            'toml',
+            'velocity',
+            'vim',
+            'php',
+            'phpdoc',
+            'xml',
+            'xml-doc',
+            'csharp',
+            'md',
+            'java',
+            'razor',
+          ],
+        },
+        splitio: {
+          // Mocked features only used when in localhost mode
+          // https://help.split.io/hc/en-us/articles/360020448791-JavaScript-SDK#localhost-mode
+          features: {
+            free_account_button_color: {
+              treatment: 'off',
+            },
+          },
+          core: {
+            authorizationKey: process.env.SPLITIO_AUTH_KEY || 'localhost',
+          },
+          debug: false,
+        },
+        newrelic: {
+          configs: {
+            development: {
+              instrumentationType: 'proAndSPA',
+              accountId: '10956800',
+              trustKey: '1',
+              agentID: '35094418',
+              licenseKey: 'NRJS-649173eb1a7b28cd6ab',
+              applicationID: '35094418',
+              beacon: 'staging-bam-cell.nr-data.net',
+              errorBeacon: 'staging-bam-cell.nr-data.net',
+            },
+            production: {
+              instrumentationType: 'proAndSPA',
+              accountId: '10956800',
+              trustKey: '1',
+              agentID: '35094662',
+              licenseKey: 'NRJS-649173eb1a7b28cd6ab',
+              applicationID: '35094662',
+              beacon: 'staging-bam-cell.nr-data.net',
+              errorBeacon: 'staging-bam-cell.nr-data.net',
+            },
+          },
+        },
+        tessen: {
+          tessenVersion: '1.14.0',
+          product: 'DOC',
+          subproduct: 'TDOC',
+          segmentWriteKey: 'AEfP8c1VSuFxhMdk3jYFQrYQV9sHbUXx',
+          trackPageViews: true,
+          pageView: {
+            eventName: 'pageView',
+            category: 'DocPageView',
+            getProperties: ({ location, env }) => ({
+              path: location.pathname,
+              env: env === 'production' ? 'prod' : env,
+            }),
+          },
+        },
+        signup: {
+          environment: process.env.ENVIRONMENT || 'staging',
+          signupUrl:
+            process.env.SIGNUP_URL ||
+            'https://signup-receiver.staging-service.newrelic.com',
+          reCaptchaToken:
+            process.env.RECAPTCHA_TOKEN ||
+            '6LdMFd8UAAAAAApWFzm8YCyuGCUfg57U1WvqVYqC',
+        },
```

Comment:
since this includes bump for theme, it includes removing standalone signup in favor of link to storefront.

we'll want to remove this recaptcha config for now so recaptcha logo doesn't show up in lower right